src/Microsoft.TestPlatform.ObjectModel/Client/TestPlatformOptions.cs

```diff
@@ -34,5 +34,11 @@ public class TestPlatformOptions
         /// </summary>
         [DataMember]
         public bool CollectMetrics { get; set; }
+
+        /// <summary>
+        ///  Gets or sets whether default adapters should be skipped or not.
+        /// </summary>
+        [DataMember]
+        public bool SkipDefaultAdapters { get; set; }
```

Comment:
>SkipDefaultAdapters [](start = 20, length = 19)

Nit: rename to SkipDefaultExtensions?

Reply:
We are skipping default adapters only and not default extensions. So as per discussion with Shiva and Pratap, made it SkipDefaultAdapters